src/drafts/MarkdownEditor/_useListEditing.ts

```diff
@@ -60,9 +60,9 @@ export const parseListItem = (line: string): ListItem | null => {
 }
 
 export const listItemToString = (item: ListItem) =>
-  `${item.leadingWhitespace}${typeof item.delimeter === 'number' ? `${item.delimeter}.` : item.delimeter}${
-    item.middleWhitespace
-  }${item.taskBox || ''} ${item.text}`
+  typeof item.delimeter === 'number'
+    ? `${item.leadingWhitespace}${`${item.delimeter}.`} ${item.text}`
```

Comment:
I think the delimiter is the dot, so this would add an extra dot:

```suggestion
    ? `${item.leadingWhitespace}${item.delimeter} ${item.text}`
```

Reply:
The delimiter is the number. The dot was there before as well. There is a test covering this

Reply:
Oh yeah, definitely see that now. That's what I get for reviewing from mobile 😄